test/pstl_offload/headers/headers.standard/utilities/version.pass.cpp

```diff
@@ -0,0 +1,15 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <version>
+#include "support/utils.h"
+
+int main() {
+    return TestUtils::done();
```

Comment:
It is definitely good to check something but the big question is what to check:)
`<version>` only defines feature test macros and there is no macro that for sure would be defined for C++17 and later because `<version>` is strictly speaking C++20 header.

Reply:
We can check some C++14 feature test macro, but we need to leave a comment that we are relying on the implementation details of C++ standard libraries that they've add this macro simultaneously with `<version>` header and it exists despite `-std=` compiler flag.

Reply:
Done